zeppelin-web/src/app/interpreter/interpreter.css

```diff
@@ -109,3 +109,696 @@
   color: #000;
   opacity: .5;
 }
+
+/**
+ * github flavored markdown style
+ */
```

Comment:
Are following CSS code blocks distributed under zeppelin-distribution/src/bin_license/licenses/LICENSE-github-markdown-css-2.4.0? Then the license need to added in /LICENSE and /licences directory under root and removed from bin_license. Because following codes wil be packaged into source release. 

And if we can create separate CSS file for it, that would be less confusing than different licences in a single file.